ios/FluentUI/Card Nudge/CardNudgeTokens.swift

```diff
@@ -16,121 +16,151 @@ import SwiftUI
-        tokens._outlineColorOverride = outlineColor
-        tokens._outlineWidthOverride = outlineWidth
-        tokens._subtitleTextColorOverride = subtitleTextColor
-        tokens._textColorOverride = textColor
-        tokens._verticalPaddingOverride = verticalPadding
-
-        return tokens
+        if let accentColor = accentColor {
+            self.accentColor = accentColor
+        }
+        if let accentIconSize = accentIconSize {
+            self.accentIconSize = accentIconSize
+        }
+        if let accentPadding = accentPadding {
+            self.accentPadding = accentPadding
+        }
+        if let backgroundColor = backgroundColor {
+            self.backgroundColor = backgroundColor
+        }
+        if let buttonBackgroundColor = buttonBackgroundColor {
+            self.buttonBackgroundColor = buttonBackgroundColor
+        }
+        if let buttonInnerPaddingHorizontal = buttonInnerPaddingHorizontal {
+            self.buttonInnerPaddingHorizontal = buttonInnerPaddingHorizontal
+        }
+        if let circleRadius = circleRadius {
+            self.circleRadius = circleRadius
+        }
+        if let circleSize = circleSize {
+            self.circleSize = circleSize
+        }
+        if let cornerRadius = cornerRadius {
+            self.cornerRadius = cornerRadius
+        }
+        if let horizontalPadding = horizontalPadding {
+            self.horizontalPadding = horizontalPadding
+        }
+        if let iconSize = iconSize {
+            self.iconSize = iconSize
+        }
+        if let interTextVerticalPadding = interTextVerticalPadding {
+            self.interTextVerticalPadding = interTextVerticalPadding
+        }
+        if let mainContentVerticalPadding = mainContentVerticalPadding {
+            self.mainContentVerticalPadding = mainContentVerticalPadding
+        }
+        if let minimumHeight = minimumHeight {
+            self.minimumHeight = minimumHeight
+        }
+        if let outlineColor = outlineColor {
+            self.outlineColor = outlineColor
+        }
+        if let outlineWidth = outlineWidth {
+            self.outlineWidth = outlineWidth
+        }
+        if let subtitleTextColor = subtitleTextColor {
+            self.subtitleTextColor = subtitleTextColor
+        }
+        if let textColor = textColor {
+            self.textColor = textColor
+        }
+        if let verticalPadding = verticalPadding {
+            self.verticalPadding = verticalPadding
+        }
     }
 
-    fileprivate var _accentColorOverride: ColorSet?
-    var accentColor: ColorSet { _accentColorOverride ?? globalTokens.brandColors[.shade20] }
+    // Required state value
+    let style: MSFCardNudgeStyle
+
+    // MARK: - Design Tokens
 
-    fileprivate var _accentIconSizeOverride: CGFloat?
-    var accentIconSize: CGFloat { _accentIconSizeOverride ?? globalTokens.iconSize[.xxSmall] }
+    lazy var accentColor: ColorSet = globalTokens.brandColors[.shade20]
```

Comment:
These tokens won't be visible publicly for client apps to query them as read only?

The computed properties approach saves all those if lets in the initializer.
Was it changed because of any size impact?

Reply:
1. Correct. I don't know of any scenario where anyone would want these to be readable. We can make them public if a specific customer need arises, though at great impact to our binary size.
2. Exactly! This conversion saved us ~40kB from the previous iteration. Personally I'm happy to trade slightly messier code for such a massive improvement (think about how much that'll save us when we convert `Button`!)